scripts/ci/upload-universe.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
```

Comment:
While I appreciate this being resritten in js, we didnt really win anything without proper implementation an testing. That said, what about this being a semantic-release plugin and adding some tests? I’m fine with a follow-up Task.

Reply:
Let's please do a follow-up task. It was meant as a quick fix for a urgent problem and I didn't expect the work to blow up that much